LibSerialize.lua

```
@@ -120,6 +120,27 @@ end
 
     Calls `SerializeEx(opts, ...)` with the default options (see below)
 
+* **`LibSerialize:SerializeAsyncEx(callback, opts, ...)`**
+
+    Arguments:
+    * `callback`: a function called when serialization is complete
+      `callback` is called with the serialized string as its argument, or nil on failure
```

Remove this, you have the explicit section below.

The serialization API didn't have a concept of a non-error()ing failure mode before - can you propagate any failure from within the coroutine and "rethrow" it?